doozer/doozerlib/cli/images_streams.py

```diff
@@ -807,6 +807,8 @@ def search_issues(query):
             'labels': ['art:reconciliation', f'art:package:{image_meta.get_component_name()}'],
             'versions': [{'name': release_version}],  # Affects Version/s
             'customfield_12319940': [{'name': Model(runtime.gitdata.load_data(key='bug').data).target_release[-1]}],  # customfield_12319940 is Target Version in jira
+            'customfield_12320850': ['Release Note Not Required'],  # customfield_12320850 is Release Notes Type in JIRA
+            'customfield_12317313': ['NA'],  # customfield_12317313 is Release Notes Text in JIRA
```

Comment:
I think this is the correct value, since the field type seems to be string
```suggestion
            'customfield_12317313': 'NA',  # customfield_12317313 is Release Notes Text in JIRA
```